src/main/resources/camel/odoo-patient-handler.xml

```diff
@@ -30,7 +30,7 @@
                 <choice>
                     <when>
                         <!-- Only process events for patients with customer records in Odoo or if we MUST create one -->
-                        <simple>${exchangeProperty.patient-odoo-id} != null || ${exchangeProperty.createCustomerIfNotExist} == true</simple>
+                        <simple>${exchangeProperty.patient-odoo-id} != null || ${exchangeProperty.createCustomerIfNotExist} == true || ${properties:create.customer.if.not.exist:null} == true</simple>
```

Comment:
Default value should be false instead of null

Reply:
Thanks @wluyima, I've added this change.